css/main.css

```diff
@@ -1,3 +1,391 @@
+}
+hr {
+    border: 1px solid red;
+    width: 30px;
+}
+.about-img {
+    padding: 0; !important
+}
+
+.mob-img {
+    display: none;
+}
+.products-2nd .grid-img {
+    display: none;
+}
+input {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 8px 0;
+    display: inline-block;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box; 
+}
+.name-form {
+    text-align: center;
+    color: white;
+}
+.email-form {
+    text-align: center;
+    color: white;
+}
+
+form {
+    margin-top: 20px;
+    width: 100%;
+    padding: 20px;
+}
+.section-form {
+    margin-top: 100px;
+    background-image: url(/img/form-back.jpeg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+}
+.section-form h2 {
+    padding: 30px;
+   color: white;
+}
+textarea {
+    width: 100%;
+}
+.sub-botton {
+    background-color: red;
+    border: none;
+    color: white;
+    font-size: 1em;
+}
+.section-footer {
+    margin-top: 80px;
+    background-color: rgba(0,0,0, .1);
+    padding: 40px;
+}
+.links-footer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.links-footer a {
+  text-decoration: none;
+  color: black;
+}
+
+.links-footer a:hover {
+    text-decoration: none;
+    color: white;
+  }
+.subs {
+    margin-top: 20px;
+}
+.subs-form {
+    display: flex;
+}
+.sub-botton-footer {
+    color: red;
+}
+.sub-botton-footer:hover {
+    color: white;
+    background-color: red;
+}
+.contact-icons-footer  {
+    margin-top: 60px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.contact-icons-footer .fa {
+   width: 30px;
+   margin-left: 40px;
+}
+.copy {
+    margin-top: 60px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+
+
+/*MEDIA*/
+
+@media screen and (min-width: 1024px) {
+    .header .menu-icon {
+        display: none;
+    }
+    .header {
+        position: fixed;
+        background-color: white;
+        box-shadow: none;
+        z-index: 1;
+
+    }
+    .header .menu {
+        overflow: visible;
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-start;
+        align-items: center;
+        left: 200px;
+        margin-top: 35px;
+    }
+    .header li a {
+        color: black;
+    }
+    .header li {
+        border: none;
+    }
+    .header li a {
+        color: black;
+    }
+
+    .main-bg {
+        position: relative;
+    }
+    .centered {
+        top: 20px;
+        position: absolute;
+        font-size: 1.5em;
+    }
+    .col-12 {
+        width: 30%;
+    }
+    .fa {
+        margin-right: 40px;
+    }
+    .contact-icons .mobile {
+        visibility: visible;
+    }
+   .section-about .col-6 {
+       width: 50%;
+   }
+   .section-about .col-3 {
+    width: 25%;
+    }
+    .products .col-4 {
+        width: 33.333333%;
+    }
+    .discount .col-6 {
+        width: 50%;
+    }
```

Comment:
Never set your own widths for `col-...` classes. That's the whole reason they exist - they have already been written!